docs/document/docs/guides/in-office-day-list-customization.md

```diff
@@ -226,10 +227,13 @@ Add event listener for FieldGroup to change the tooltip message when FieldGroup
 
 
 ### Using Two Types of Tooltip
-You may have noticed that when creating a Tooltip for a button, I will introduce a new property, describeChild. 
+You may have noticed that when creating a Tooltip for a button, I will introduce a new property `describeChild`.
 This property is set to false by default.<br/>
-You can effectively improve the accessibility and usability of your application with the describeChild property.<br/>
-Now, let's enable the screen reader and try moving the focus to the button to see what the screen reader reads.
+You can effectively improve the accessibility and usability of your application with the `describeChild` property.<br/>
+Now, let's enable the [screen reader](https://support.apple.com/en-us/guide/voiceover/vo2682/mac) and try moving the focus to the button to see what the screen reader reads.<br/>
+
+#### Tooltip represents the purpose of the element
+If the Tooltip represents the purpose of the element itself (provides a label for the element it is attached to). You need to set `describeChild` to false.
 ```javascript
   const tooltipForButton = new Kuc.Tooltip({
     title: 'Submit changes or additions for this entry.',
```

Comment:
This is description or explanation of the target container, so describeChild should be set to true.
So this is not a good example for describeChild: false case.
Please consider another one.